es-es/perl6-es.html.markdown

```diff
@@ -1,19 +1,19 @@
 ---
-category: language
 language: perl6
-filename: learnperl6-es.p6
+filename: perl6-es.p6
 contributors:
     - ["vendethiel", "http://github.com/vendethiel"]
     - ["Samantha McVey", "https://cry.nu"]
 translators:
 	- ["Luis F. Uceta", "https://github.com/uzluisf"]
```

Comment:
Remove the whitespace here.

Reply:
And the travis build has been failing with I guess this whitespace error. Do check whether travis builds or not after another commit and try to figure out the source of the problem.